Framework/Source/Graphics/Material/Material.cpp

```diff
@@ -332,8 +332,8 @@ namespace Falcor
 
         // Now set the textures
         std::string resourceName = std::string(varName) + ".textures.layers";
```

Comment:
Just like the `.desc.layers[0].type` isn't needed above any more, if your new reflection API is modeled on Slang's approach (where a single variable could have offsets for multiple parameter categories) then you similarly shouldn't need the `.textures.layers` here. Of course whether that works in practice is a different thing.

Reply:
It won't work in Falcor. getResourceBinding() accepts the find a variable with a resource-type. The user can call findMember() with the var-name and extract the resources from it, which is less convenient.